src/main/java/bitflow4j/script/registry/AbstractRegisteredStep.java

```diff
@@ -8,6 +8,7 @@
 public abstract class AbstractRegisteredStep {
 
     public final String name;
+    public String conventionName;
```

Comment:
Please rename conventionName to stepName, and name to className.
Sorry for nitpicking.